docs/admin_api/rooms.md

```diff
@@ -13,6 +13,9 @@ The following query parameters are available:
 * `order_by` - The method in which to sort the returned list of rooms. Valid values are:
   - `alphabetical` - Rooms are ordered alphabetically by room name. This is the default.
   - `size` - Rooms are ordered by the number of members. Largest to smallest.
+  - `name`, `canonical_alias`, `joined_members`, `joined_local_members`, `version`, `creator`,
+    `encryption`, `federatable`, `public`, `join_rules`, `guest_access`, `history_visibility`,
+    `state_events` - ordered by this column
```

Comment:
These should be a bit more detailed, in particular it seems that some of them sort largest first, some alphabetically, etc.